openedx/adg/admin_dashboard/admin_settings/helpers.py

```diff
@@ -134,3 +139,37 @@ def get_title_for_action_completion(action_type):
         UNDO_INVITATION: PERMISSIONS_RESTORED
     }
     return action_type_to_title_map.get(action_type, '')
+
+
+def get_admin_details(admin):
+    """
+    Returns the admin name, permission (ADG Admin or BU Admin) and business line in case of BU Admin.
+
+    Arguments:
+        admin (User): admin for which the details are to be fetched
+
+    Returns:
+        dict: dictionary containing admin details
+    """
+    name = admin.profile.name
+    image = get_profile_image_urls_for_user(admin).get('medium')
+
+    if is_superuser_or_adg_admin(admin):
+        permission = Text(_('{name}, {span_start}ADG Admin{span_end}')).format(
+            name=name,
+            span_start=HTML('<span class="admin-designation">'),
+            span_end=HTML('</span>'),
+        )
+    else:
+        business_unit = BusinessLine.business_lines_for_user(admin).first().title
```

Comment:
Perhaps we should refactor this `business_lines_for_user` function to only return one business line?

Reply:
@aightmunam actually we can add multiple business lines for an admin so this method would return all those.

Reply:
Didn't Andrew say that that would never happen? @annas-usman